packages/common/src/types.ts

```diff
@@ -2,14 +2,12 @@
  * Apache-2.0 */
 
 import {
-  IItem,
-  IUser,
   IGroup,
   IPolygon,
   ISpatialReference,
```

Comment:
@rweber-esri good catch - at rest-js 4.x I think the types package is gone... and everything is exported from implementation packages... so I generally import from -portal if possible - there are a few types related to feature services that only exist in -rest-types for 3.x, but otherwise... use portal if possible

Reply:
Good to know, thanks for the additional context.